docs/COMPATIBILITY.md

```diff
@@ -25,16 +25,25 @@ Increases of minimum requirements are indicated in bold.
 	</tr>
 	<tr>
 		<th>5.0.x</th>
-		<td>Future version</td>
-		<td>-</td>
-		<td>-</td>
+		<td>Future release</td>
+		<td></td>
+		<td></td>
 		<td><strong>8.0.x</strong> - TBD</td>
 		<td><strong>1.39.0</strong> - TBD</td>
 		<td></td>
 	</tr>
 	<tr>
-		<th>4.1.x</th>
+		<th>4.2.x</th>
 		<td><strong>Stable release</strong></td>
+		<td>2024-07-05</td>
+		<td>2024-07-05</td>
+		<td><strong>7.4.0</strong> - 8.1.x</td>
+		<td>1.35.0 - 1.39.x</td>
```

Comment:
I suspect that at this point SMW works reasonably well with more recent MW releases.

Anyone has concrete experiences to contribute? Perhaps @krabina ?

I ran tests with recent MW versions via https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/5660

Reply:
I only use LTS versions, so no. It does work with MediaWiki 1.39 and PHP 8.2, though.

Reply:
> It does work with MediaWiki 1.39 and PHP 8.2, though.

This would be cool for Debian users. It's another story, though.

Reply:
It looks bad to release without supporting the last 3 MW versions. Perhaps we can avoid it. We do not know how many tests fail, only that the test setup itself does not work on those MW versions, which says nothing about SMWs compat.

Reply:
Basic SMW field usage works with PHP 8.1 and 8.2 for MW 1.41. So, I assume MW 1.40 will also work for you. It would be nice to get tests running in the repo, though.

I will be making the adjustments to this PR once all the information is available.

Reply:
> I only use LTS versions, so no. It does work with MediaWiki 1.39 and PHP 8.2, though.

@krabina I've got it in prod with MediaWiki 1.39.8 and PHP 8.2.21...! What issues do you have on your side ?
https://wiki.tripleperformance.fr/wiki/Sp%C3%A9cial:Version

Reply:
> It does work with MediaWiki 1.39 and PHP 8.2, though.

It looks like he actually does not have issues with PHP 8.2